core/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceConfig.java

```diff
@@ -76,6 +76,21 @@ public boolean valid() {
         return this;
     }
 
+    public ValuesSourceConfig<VS> format(final DocValueFormat format) {
```

Comment:
Should we make all the fields of this class private now we have setters/getters for them so there is only one way of accessing/modifying the values? (maybe in a followup PR?)